dash/include/dash/map/UnorderedMap.h

```diff
@@ -643,6 +611,68 @@ class UnorderedMap
     return result;
   }
 
+  iterator insert(
+    const_iterator hint,
+    const value_type & value)
+  {
+    Timer::timestamp_t ts_enter = Timer::Now(), ts_insert, ts_find, d_insert, d_find;
```

Comment:
This was actually a commit by accident. Will remove this either. It is something which does not really belong to this pull request.